README.md

```diff
@@ -1,6 +1,6 @@
 # AWS IoT Device SDK for Embedded C
 
-This tag [3.1.4](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.1.4) contains the v3 version of AWS IoT Device SDK for Embedded C. No new features will be added to this tag; instead, only bug fixes will be made and minimally tested.
+This tag [3.1.5](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.1.5) contains the v3 version of AWS IoT Device SDK for Embedded C. No new features will be added to this tag; instead, only bug fixes will be made and minimally tested.
```

Comment:
Shouldn't the bugfix information be added to CHANGELOG for all our latest fixes to v3?

Reply:
The changelog hasn't been updated in almost 3 years. It might make sense to start updating the changelog for our fixes, but I'm not sure if it makes sense to do so without also logging all of our changes since then.